xwiki/values.yaml

```diff
@@ -0,0 +1,30 @@
+# Default values for node.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  pullPolicy: IfNotPresent
+service:
+  name: node
```

Comment:
Service name could be autogenerated based on Helm deployment name - if not provided. If provided, you could pass it from here, but maybe give it a better name? :)

Reply:
Corrected this it was the name of port